parser/ast/ddl.go

```diff
@@ -1180,6 +1180,13 @@ func (n *CreateTableStmt) Accept(v Visitor) (Node, bool) {
 		}
 		n.Partition = node.(*PartitionOptions)
 	}
+	for i, option := range n.Options {
```

Comment:
Is it required by the PR? I think it is right to visit all the Options for `CreateTableStmt.Accept`. But I'm not sure why we did not do it before, maybe just because forgot it?

Reply:
Yes. The `parser_test.go` requires to visit all `Expr` to reset the offset. Without it, we cannot use the unit test utilities.